gocd/vagrant-publisher.opensuse.gocd.yaml

```diff
@@ -0,0 +1,22 @@
+format_version: 3
+pipelines:
+  Vagrant.Publish:
```

Comment:
I was actually referring to the pipeline name, which should mention `openSUSE`.

The filename can stay as it was, it might contain also SLE pipelines in the future.

Reply:
Like this?